test/fixtures/google-gax-packaging-test-app/test/gapic-v1beta1.js

```diff
@@ -177,6 +177,45 @@ describe('EchoClient', () => {
       });
     });
 
+    it('invokes pagedExpand using async iterator', async () => {
+      const client = new showcaseModule.v1beta1.EchoClient({
+        credentials: {client_email: 'bogus', private_key: 'bogus'},
+        projectId: 'bogus',
+      });
+
+      // Mock request
+      const request = {};
+
+      // Mock response
+      const nextPageToken = '';
+      const responsesElement = {};
+      const expectedResponse = {
+        nextPageToken: nextPageToken,
+        responses: responsesElement,
+      };
+
+      client._descriptors.page.pagedExpand.asyncIterate = (apiCall, request, options) => {
+        const asyncIterable = {
+          [Symbol.asyncIterator]() {
+            return {
+              async next(){
+                return Promise.resolve({done: true, value: -1});
```

Comment:
`undefined` also

Reply:
maybe implement a real logic that returns values from `expectedResponse` one by one?